src/data.js

```diff
@@ -10,8 +10,10 @@ export function create (cozy, doctype, attributes) {
     if (isV2) {
       attributes.docType = doctype
     }
-    const path = createPath(cozy, isV2, doctype)
-    return cozyFetchJSON(cozy, 'POST', path, attributes).then((resp) => {
+    const path = createPath(cozy, isV2, doctype, attributes._id)
+    const VERB = attributes._id ? 'PUT' : 'POST'
```

Comment:
That was to reflect the value of verbs, but it may not be a very clever idea. ;-)